tests/testthat/reporters/progress.txt

```diff
@@ -1,16 +1,16 @@
 ✔ |  OK F W S | Context
 
-⠏ |   0       | tests
-⠋ |   1       | tests
+⠏ |   0       | tests                                                           
+⠋ |   1       | tests                                                           
```

Comment:
Do you know why the whitespace differences happen? Is it because of changes in testthat itself?

Reply:
That's the new `strpad()` I added in case the first context is shorter than the second